raepa/dock.py

```diff
@@ -78,4 +78,4 @@ def help_database():
         """
         Display the help on database structure
         """
-        open_new(r'https://3liz.github.io/qgis-raepa-plugin/')
+        open_new(r'https://3liz.github.io/qgis-raepa-plugin/database')
```

Comment:
```suggestion
        open_new(r'https://3liz.github.io/qgis-raepa-plugin/')
```

Je pense que tu peux plutôt changer le nom de la fonction en `help_plugin`. C'était avant quand on avait **que** la documentation de la BDD. Maintenant, on préfère ouvrir le site mkdocs avec l'ensemble de la documentation.